src/ResourceManager/KeyVault/Commands.KeyVault/Commands/RemoveAzureKeyVaultAccessPolicy.cs

```diff
@@ -109,25 +205,44 @@ public class RemoveAzureKeyVaultAccessPolicy : KeyVaultManagementCmdletBase
             ParameterSetName = ByObjectId,
             ValueFromPipelineByPropertyName = true,
             HelpMessage = "Specifies the ID of application that a user must.")]
+        [Parameter(Mandatory = false,
+            ParameterSetName = InputObjectByObjectId,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Specifies the ID of application that a user must.")]
         public Guid? ApplicationId { get; set; }
 
         [Parameter(Mandatory = false,
             ParameterSetName = ForVault,
             ValueFromPipelineByPropertyName = true,
             HelpMessage = "If specified, disables the retrieval of secrets from this key vault by the Microsoft.Compute resource provider when referenced in resource creation.")]
-        public SwitchParameter EnabledForDeployment { get; set; }
+        [Parameter(Mandatory = false,
+            ParameterSetName = InputObjectForVault,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "If specified, disables the retrieval of secrets from this key vault by the Microsoft.Compute resource provider when referenced in resource creation.")]
+        [Alias("EnabledForDeployment")]
+        public SwitchParameter DisableDeployment { get; set; }
```

Comment:
I have some concerns regarding this renaming (as well as the subsequent one, re: template deployment). These are actually vault properties, and the name was chosen specifically to match their corresponding vault properties. The renaming not only diverges from this implicit tie-in, but also reverses its semantics: the user must now negate this value to understand their effect.
Please consider reverting this specific change.